zio/tzngio/reader.go

```diff
@@ -210,3 +212,9 @@ func (r *Reader) parseValue(line []byte) (*zng.Record, error) {
 
 	return zng.NewRecordCheck(typ, raw)
 }
+
+func ReadAll(str string) (zbuf.Array, error) {
```

Comment:
This feels odd, partly because it isn't really in keeping with ioutil.ReadAll. If you want something like it, I suggest adding zbuf.ReadAll and tzngio.NewReaderString.

Reply:
I agree it should be in zbuf. Also should have a context. I still thinks its cumbersome to have to write `zbuf.ReadAll(tzngio.NewReader(resolver.NewContext(), strings.NewReader(str)))`, for something we end up doing a lot in tests.

Perhaps we could add this a function in some kind of test utility?